src/containers/DataDisplay/DataViewTable.tsx

```diff
@@ -214,14 +222,3 @@ const isSorted = (sortColumn: string | undefined, apiQueries: DataViewTableAPIQu
   if (sortColumn !== apiQueries.orderBy) return undefined
   return apiQueries.ascending === 'true' ? 'ascending' : 'descending'
 }
-
-const getSortColumnInfo = (
-  dataViewTable: DataViewsTableResponse | undefined,
-  sortBy: string
-): SortColumn => {
-  if (!sortBy || !dataViewTable) return null
-  const [fieldName, direction] = sortBy.split(':')
-  const ascending = direction === 'desc' ? false : true
-  const column = dataViewTable.headerRow.find((col) => col.columnName === fieldName)
-  return { title: column?.title || '', ascending }
-}
```

Comment:
Unrelated, but this was dead code -- I think it's leftover from my work on the the Table sorting, but I guess I didn't end up using it.